web-components/src/components/floating-modal/FloatingModal.ts

```diff
@@ -312,7 +312,7 @@ export namespace FloatingModal {
                   ${!this.minimize ? html` <md-button
                     color="color-none"
                     class="md-floating__resize"
-                    aria-label="${this.resizeAriaLabel}"
+                    aria-label="${this.full ? 'Fullscreen Resize Modal' : 'Halfscreen Resize Modal'}"
```

Comment:
is this maximize and restore?
change the text

'Maximize the modal' : 'Restore the maximised modal', But this should not hard coded inside the component. the localized message should be passed by client code e.g. desktop